SoftLayer/CLI/vlan/create.py

```diff
@@ -0,0 +1,40 @@
+"""Order/create a dedicated server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(epilog="See 'slcli server create-options' for valid options.")
+@click.option('--hostname', '-H', required=True, prompt=True, help="Host portion of the FQDN")
```

Comment:
@click.option('--vlan-name', '-v', required=false, prompt=True, help="vlan help")

As the VLAN name is optional in UI, better to use the same logic here too.